medicai/utils/registry.py

```
@@ -0,0 +1,53 @@
+                f"Backbone '{name}' not found. Available: {list(self._registry.keys())}"
+            )
+        return self._registry[key]
+
+    def get(self, name: str):
+        return self.get_entry(name)["class"]
+
+    def create(self, name: str, **kwargs):
+        cls = self.get(name)
+        return cls(**kwargs)
+
+    def list(self, family: Union[str, List[str]] = None, details: bool = False):
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)